src/subclass/container.rs

```diff
@@ -24,7 +24,7 @@ pub trait ContainerImpl: ContainerImplExt + WidgetImpl + 'static {
         self.parent_check_resize(container)
     }
 
-    fn set_focus_child(&self, container: &Container, widget: &Widget) {
+    fn set_focus_child(&self, container: &Container, widget: Option<Widget>) {
```

Comment:
```suggestion
    fn set_focus_child(&self, container: &Container, widget: Option<&Widget>) {
```

And similar for the other places